src/views/Rules/ListRules/index.jsx

```diff
@@ -476,7 +628,10 @@ function ListRules(props) {
           className={classes.ruleCard}
           key={rule.rule_id}
           rule={rule}
+          readOnly={username === ''}
```

Comment:
Let's not remove action buttons when the user is not logged in. Instead, lets disable them. Disabled action buttons will let the user know that they need to login in order to use them.